src/game.cpp

```diff
@@ -4166,12 +4166,9 @@ void game::debug()
                         }
 
                         if( bp_ptr != nullptr ) {
-                            std::string raw_input = query_string( "Set the stat to? Currently: %d", *bp_ptr );
-                            if( raw_input.empty() ) { // ESC is pressed
-                                break;
-                            }
-                            int value = atoi( raw_input.c_str() );
-                            if( value >= 0 ) {
+                            int value;
+                            bool got_value = query_int( value, "Set the stat to? Currently: %d", *bp_ptr );
```

Comment:
You don't need the temporary. You can do `if( query_int(`. This is the same idiom used by `JsonObject::read`